static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -139,15 +144,15 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
     hasProperlyLoadedConditions: boolean
   ): [string, string | React.ReactElement][] {
     const customizedAlertOption: [string, React.ReactNode] = [
-      Actions.CUSTOMIZED_ALERTS.toString(),
+      RuleAction.CUSTOMIZED_ALERTS.toString(),
       <CustomizeAlertsGrid
-        key={Actions.CUSTOMIZED_ALERTS}
+        key={RuleAction.CUSTOMIZED_ALERTS}
```

Comment:
updated the name just because I had to use it in another file, so it was easier to identify what this is about